src/createServer.js

```diff
@@ -1,8 +1,82 @@
 'use strict';
 
+const http = require('node:http');
+const path = require('path');
+const fs = require('fs');
+
+const CONTENT_TYPES = {
+  '.html': 'text/html',
+  '.css': 'text/css',
+  '.js': 'application/javascript',
+  '.json': 'application/json',
+  '.jpg': 'image/jpeg',
+  '.jpeg': 'image/jpeg',
+  '.png': 'image/png',
+  '.gif': 'image/gif',
+  '.svg': 'image/svg+xml',
+  '.txt': 'text/plain',
+};
+
+const getContentType = (filePath) => {
+  const extname = path.extname(filePath).toLowerCase();
+
+  return CONTENT_TYPES[extname] || 'application/octet-stream';
+};
+
+const sendErrorResponse = (res, code, message) => {
+  res.statusCode = code;
+  res.end(message);
+};
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'text/plain');
+
+    const { pathname } = new URL(req.url, `http://${req.headers.host}`);
+
+    if (pathname.includes('//')) {
+      return sendErrorResponse(
+        res,
+        404,
+        'Double slashes are not allowed in the URL.',
+      );
+    }
+
+    if (pathname === '/file/' || pathname === '/file') {
+      return sendErrorResponse(
+        res,
+        200,
```

Comment:
The status code for this response is set to 200, which typically indicates a successful request. However, since this is an error message indicating a misuse of the path, it might be more appropriate to use a 400 status code to indicate a bad request.